go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/rook/rook
 
-go 1.23.0
+go 1.24.1
```

Comment:
```suggestion
go 1.24.5
```

Why not latest version?

Reply:
@subhamkrai and which toolchain version?